### DISCUSSION TITLE
是新角色！卡吕冬之子的一员

### DISCUSSION THREAD

**Discussion by OoTyleroO** [share121/inter-knot#1651] [其他]
![b3250e1baabd588c5d1e7d83ea98095b1636034895](https://github.com/user-attachments/assets/5d12fbb9-a91e-4fa3-83c8-7129ba933dee)
![1723364487996](https://github.com/user-attachments/assets/6a8b6b30-5be7-4a35-9213-431fcd6dad01)
代理人档案 | 柏妮思·怀特（火•异常）

「你看起来不太热烈呢，要『燃』一下吗~」
柏妮思·怀特

「味道不错吧~这杯全新口味的燃油饮可是加入了从城里带回来的特别配料噢。」
「什么？等会儿要回车上打个盹，不想喝完太兴奋是吗？」
「这么说来…你有车啊！我这里也有独家调配的超节能机用燃油，拿去试试吧！」

中文CV——葛子瑞
日文CV——高桥花林
大家怎么看

**Comment by cooperatior** [comment]
我在看一眼就......飞起来了

**Comment by JustOldJohn** [comment]
**火系异常好耶**

**Comment by JustOldJohn** [comment]
不知道莱特什么时候来呢

**Comment by OoTyleroO** [comment]
如果莱特4星的话，明天应该就出来了

**Comment by lyk758811** [comment]
Big 可莉？（幻视

**Comment by szlx2333** [comment]
居然是火异常么，这不是可以和露西拍档
